app/server/database/database.py

```diff
@@ -13,10 +15,26 @@
 user_collection = database.get_collection('users_collection')
 admin_collection = database.get_collection('admins')
 
-async def add_admin(admin_data: dict) -> dict:
-    admin = await admin_collection.insert_one(admin_data)
-    new_admin = await admin_collection.find_one({"_id": admin.inserted_id})
-    return admin_helper(new_admin)
+async def add_admin(admin_data: dict) -> Optional[dict]:
+    # add admin if email not present already
+    # param admin_data : dict -> contains keys fullname, email, password
+    # return
+    #       dict -> return user details
+    #       None -> if email was already present
+    admin = await admin_collection.find_one({'email':admin_data.get('email')})
+    if admin == None:
+        admin = await admin_collection.insert_one(admin_data)
```

Comment:
@PranavBakre @vigu184 
The _insert_one()_ method returns a _InsertOneResult_ object, which has a property, _inserted_id_, that holds the id of the inserted document. ( [check here](https://www.w3schools.com/python/python_mongodb_insert.asp) )
so in order to return the data of added admin, executing _find_one()_ using _inserted_id_ as parameter is necessary